cli/tests/common.rs

```diff
@@ -16,18 +16,14 @@ use tempfile::TempDir;
 pub const API_URL: &str = "https://api.staging.phylum.io";
 const PROJECT_NAME: &str = "integration-tests";
 
+#[derive(Default)]
 enum Cwd {
     Path(PathBuf),
     TempDir,
+    #[default]
```

Comment:
Isn't this a 1.65.0 feature? Our CLI is 1.64.0+ min version.

Though I guess that's fine for tests.

Reply:
Good catch. Are you opposed to bumping the MSRV to 1.65?

Reply:
Not at all.